autotest/test_order3_command_chat.py

```diff
@@ -0,0 +1,155 @@
+    def test_chat_Baichuan2_7B_Chat(self, config, case_config, usercase):
+        result = run_command_line_test(config, usercase,
+                                       case_config.get(usercase),
+                                       'Baichuan2-7B-Chat')
+        assert result.get('success'), result.get('msg')
+
+    @pytest.mark.Baichuan2_7B_Chat
+    @allure.story('Baichuan2-7B-Chat-inner-w4')
+    @pytest.mark.parametrize('usercase', getList())
+    def test_chat_Baichuan2_7B_Chat_inner_w4(self, config, case_config,
+                                             usercase):
+        result = run_command_line_test(config, usercase,
+                                       case_config.get(usercase),
+                                       'Baichuan2-7B-Chat-inner-w4')
+        assert result.get('success'), result.get('msg')
+
+    @pytest.mark.CodeLlama_7b_Instruct_hf
+    @allure.story('CodeLlama-7b-Instruct-hf')
+    @pytest.mark.parametrize('usercase', getList())
+    def future_test_chat_CodeLlama_7b_Instruct_hf(self, config, case_config,
+                                                  usercase):
+        result = run_command_line_test(config, usercase,
+                                       case_config.get(usercase),
+                                       'CodeLlama-7b-Instruct-hf')
+        assert result.get('success'), result.get('msg')
+
+    @pytest.mark.CodeLlama_7b_Instruct_hf
+    @allure.story('CodeLlama-7b-Instruct-hf-inner-w4')
+    @pytest.mark.parametrize('usercase', getList())
+    def future_test_chat_CodeLlama_7b_Instruct_hf_inner_w4(
+            self, config, case_config, usercase):
+        result = run_command_line_test(config, usercase,
+                                       case_config.get(usercase),
+                                       'CodeLlama-7b-Instruct-hf-inner-w4')
+        assert result.get('success'), result.get('msg')
+
+    @pytest.mark.llama_2_7b_chat
+    @allure.story('llama-2-7b-chat')
+    @pytest.mark.parametrize('usercase', getList())
+    def test_chat_llama_2_7b_chat(self, config, case_config, usercase):
+        result = run_command_line_test(config, usercase,
+                                       case_config.get(usercase),
+                                       'llama-2-7b-chat')
+        assert result.get('success'), result.get('msg')
+
+    @pytest.mark.llama_2_7b_chat
+    @allure.story('llama-2-7b-chat-inner-w4')
+    @pytest.mark.parametrize('usercase', getList())
+    def test_chat_llama_2_7b_chat_inner_w4(self, config, case_config,
+                                           usercase):
+        result = run_command_line_test(config, usercase,
+                                       case_config.get(usercase),
+                                       'llama-2-7b-chat-inner-w4')
+        assert result.get('success'), result.get('msg')
+
+
+def run_command_line_test(config, case, case_info, model):
+    result, chat_log, msg = commandLineTest(config, case, case_info, model,
```

Comment:
we should make the naming style consistent: Change the camel case?  `commandLineTest` -> `command_line_test`?